observations/obs_converters/quikscat/QuikSCAT.rst

```diff
@@ -49,44 +49,46 @@ Programs
 --------
 
 There are several programs that are distributed from the JPL www-site,
-ftp://podaac.jpl.nasa.gov/pub/ocean_wind/quikscat/L2B/sw/; we specifically started from the Fortran file
-`read_qscat2b.f <ftp://podaac.jpl.nasa.gov/pub/ocean_wind/quikscat/L2B/sw/FORTRAN/read_qscat2b.f>`__ and modified it to
-be called as a subroutine to make it more similar to the rest of the DART framework. The original ``Makefile`` and
-``read_qscat2b.f`` are included in the DART distribution in the ``DART/observations/quikscat`` directory. You will have
-to modify the ``Makefile`` to build the executable.
+ftp://podaac.jpl.nasa.gov/pub/ocean_wind/quikscat/L2B/sw/; we modified the Fortran file
```

Comment:
Is this supposed to be an active link?

Reply:
Yes it opens the file manager for ftp transfer.